### ISSUE TITLE
Graph API gets Facebook error 5

### ISSUE THREAD

**Issue by hsmith825** [Wizcorp/phonegap-facebook-plugin#906]
I have used this app for a long time and had a number of IOS releases with the code.  All of a certain I started getting this error from the graph api.  Here is what I know about the problem now. I build successfully and successfully login with the Facebook credentials. I then use the graph api as follows: facebookConnectPlugin.api([my id] +"/?fields=name,picture,email,first_name, age_range,currency,languages",["user_birthday"],function (response) {my code}, function (error) {
alert("Failed: " + error);}.
The error I get from Facebook is Failed. The operation couldn't be completed. (com.facebook.sdk error5.). The operation couldn’t be completed. (com.facebook.sdk error 5.). This worked until recently with no changes to the Facebook code.  I am only using the login sequence which has complete and this graph API.  Something changed in the plugin orr at Facebook and not in my code.  
Any suggestions are appreciated?

**Comment by vtellez** [comment]
Same problem here! Did you find any solution?

**Comment by hsmith825** [comment]
Yes I did.  I discovered that I had a problem in a further call requesting user permissions.  One of the permissions was no longer available without Facebook review.

**Comment by vtellez** [comment]
Thanks for your answer.

It's seem that i need to review my app to facebook process for "Publish_actions" permission, but when i try to send to facebook, i get the next warning:

It looks like you haven't made any API requests to access content with the publish_actions permission in the last 30 days
